### DISCUSSION TITLE
unable to install ganache in my system

### DISCUSSION THREAD

**Discussion by decentralized-86** [ConsenSys-archive/truffle#5439] [Installation]
![Screenshot (44)](https://user-images.githubusercontent.com/109427449/184621960-fc57a75e-1a65-43f8-9068-bdb3fe9e2be0.png)

**Comment by apdan510** [comment]
Hey @decentralized-86 ,

Try this Ganache-ui exe installer instead if you running into issues installing the .appx installer. 

https://github.com/trufflesuite/ganache-ui/releases/download/v2.5.4/Ganache-2.5.4-win-setup.exe

Kind regards,
Alex | Consensys
